_episodes/access.md

```diff
@@ -39,16 +39,102 @@ Next to large-scale general-use repositories such as [Zenodo](https://zenodo.org
 
 > ## Stages of accessibility
 >
-> You submitted a paper, and the reviewer asks you to make your data accessible. Hence,
+> You submitted a paper, and the reviewer asks you to make your data accessible. You consider your options,
 >
-> - You make a note in your paper that data can be requested by sending you an email
-> - You provide the data as supplementary information to your paper
-> - You upload the data to some cloud storage and put a shareable link in your paper
-> - You upload the data to a university drive and request a DOI from the library
-> - You upload the data to a generic repository such as zenodo
-> - You upload the data to a domain-specific repository such as [Hydroshare](https://www.hydroshare.org/)
+> 1. You make a note in your paper that data can be requested by sending you an email
+> 2. You provide the data as supplementary information to your paper
+> 3. You upload the data to some cloud storage and put a shareable link in your paper
+> 4. You upload the data to a university drive and request a DOI from the library
+> 5. You upload the data to a generic repository such as zenodo
+> 6. You upload the data to a domain-specific repository such as [Hydroshare](https://www.hydroshare.org/)
 >
-> Describe which of these are adhering to the FAIR principles.
+> Describe which of these are adhering to the [FAIR principles](https://www.go-fair.org/fair-principles/).
+>
+>> ## Solution
+>> <table style="width:50%">
```

Comment:
I like the structured approach here, but it would be nice to also have a very short overall written verdicts

Reply:
I tried that, but it's not flexible and it does not render in the nested boxes.